roles/kubernetes-apps/network_plugin/contiv/tasks/main.yml

```diff
@@ -0,0 +1,118 @@
+- name: Create contiv-config ConfigMap
+  run_once: true
+  kube:
+    name: "contiv-config"
+    kubectl: "{{ bin_dir }}/kubectl"
+    filename: "{{ kube_config_dir }}/contiv-config.yaml"
+    resource: "configmap"
+    namespace: "{{ system_namespace }}"
+
+- name: Create contiv-etcd-proxy DaemonSet
+  run_once: true
+  kube:
+    name: "contiv-etcd-proxy"
+    kubectl: "{{ bin_dir }}/kubectl"
+    filename: "{{ kube_config_dir }}/contiv-etcd-proxy.yaml"
+    resource: "daemonset"
+    namespace: "{{ system_namespace }}"
+
+- name: Create contiv-netplugin DaemonSet
+  run_once: true
+  kube:
+    name: "contiv-netplugin"
+    kubectl: "{{ bin_dir }}/kubectl"
+    filename: "{{ kube_config_dir }}/contiv-netplugin.yaml"
+    resource: "daemonset"
+    namespace: "{{ system_namespace }}"
+
+- name: Create contiv-netmaster DaemonSet
+  run_once: true
+  kube:
+    name: "contiv-netmaster"
+    kubectl: "{{ bin_dir }}/kubectl"
+    filename: "{{ kube_config_dir }}/contiv-netmaster.yaml"
+    resource: "daemonset"
+    namespace: "{{ system_namespace }}"
+
+- name: Create contiv-api-proxy DaemonSet
+  run_once: true
+  kube:
+    name: "contiv-api-proxy"
+    kubectl: "{{ bin_dir }}/kubectl"
+    filename: "{{ kube_config_dir }}/contiv-api-proxy.yaml"
+    resource: "daemonset"
+    namespace: "{{ system_namespace }}"
+  when: contiv_enable_api_proxy
+
+- name: Create contiv ClusterRoleBinding
```

Comment:
I've tested it with Kargo which doesn't seem to enable rbac. When Kargo will implement rbac, this plugin would normally work.